flocker/provision/_ssh/_model.py

```diff
@@ -52,15 +52,20 @@ class Run(PRecord):
     :ivar bytes command: The command to run.
     :ivar callable log_command_filter: A filter to apply to any logging
         of the executed command.
+    :ivar bytes pre_command: A command to run before the ``command``,
+        generally to setup the environment for the command.
```

Comment:
An alternative would be to leave this as `command` and have a separate helper function somewhere called `venv_command` (or something), that takes an arbitrary command and sticks 'source bin/activate; ' in front of it.